src/reports_api/utils/helpers.py

```diff
@@ -0,0 +1,20 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from reports_api.models import db
+
+
+def find_model_from_table_name(table_name: str):
```

Comment:
Could you please check code.py in services. Find model by tablename is already achieved there.

Reply:
Updated it to use new method from utils